src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,100 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import { getMovie } from '../../api/getMovie';
+import { MovieCard } from '../MovieCard';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+type Props = {
+  onAdd:(movie: Movie) => void,
+};
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie: React.FC<Props> = ({ onAdd }) => {
+  const [title, setTitle] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [error, setError] = useState(false);
+
+  const handleTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+  };
+
+  const handleRequest = () => {
+    getMovie(title)
+      .then(movieFromServer => {
+        if (movieFromServer.Response === 'False') {
+          setMovie(null);
+          setError(true);
+        }
+
+        setMovie(movieFromServer);
+      });
+  };
+
+  const handleFocus = () => {
+    setError(false);
+  };
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
+  const handleSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+
+    if (movie?.Response === 'True') {
+      onAdd(movie);
+      setTitle('');
+    }
+  };
+
+  return (
+    <>
+      <form className="find-movie" onSubmit={handleSubmit}>
+        <div className="field">
+          <div className="control">
+            <label className="label" htmlFor="movie-title">
+              Movie title
+              <input
+                type="text"
+                id="movie-title"
+                placeholder="Enter a title to search"
+                className="input is-danger"
+                value={title}
+                onChange={handleTitleChange}
+                onFocus={handleFocus}
+              />
+            </label>
+          </div>
+
+          {error && (
+            <p className="input is-danger">
```

Comment:
```suggestion
            <p className="help is-danger">
```
Не стилізуй цей текст так само, як інпут. Це просто має бути червоний текст